cidc_api/services/ingestion.py

```diff
@@ -199,18 +203,36 @@ def wrapped(*args, **kwargs):
             **kwargs,
         )
 
-    wrapped.__name__ = f.__name__
     return wrapped
 
 
+
+@ingestion_api.route("/validate", methods=["POST"])
+@requires_auth(
+    "ingestion/validate", [CIDCRole.ADMIN.value, CIDCRole.NCI_BIOBANK_USER.value]
+)
+@upload_handler
+def validate_endpoint(
+    user: Users,
+    trial: TrialMetadata,
+    template_type: str,
+    xlsx_file: BinaryIO,
+    md_patch: dict,
+    file_infos: List[prism.LocalFileUploadEntry],
+):
+    # Validation is done within `upload_handler`
+    # so we just return ok here
+    return jsonify({"errors": []})
```

Comment:
Shouldn't this return collected validation errors?

Reply:
as this is wrapped with `upload_handler` that will return all validation errors, here we now validation has passed so no errors

Reply:
I'd make the signature `def validate_endpoint(*args, **kwargs)` to make it more explicit that none of those arguments are used.